.devcontainer/Dockerfile

```diff
@@ -77,3 +77,21 @@ RUN chown -R $USERNAME:$USERNAME /opt/fsl-imx-xwayland/5.10-hardknott/
 
 # Add access to openocd for VSCode debugging
 RUN chown -R $USERNAME:$USERNAME /opt/openocd
+
+# Fix Tizen SDK paths for new user
+RUN sed -i '/^TIZEN_SDK_DATA_PATH/d' $TIZEN_SDK_ROOT/sdk.info \
+    && echo TIZEN_SDK_DATA_PATH=/home/$USERNAME/tizen-sdk-data >> $TIZEN_SDK_ROOT/sdk.info \
+    && ln -sf /home/$USERNAME/.tizen-cli-config $TIZEN_SDK_ROOT/tools/.tizen-cli-config \
+    && : # last line
+
+USER $USERNAME
+WORKDIR /home/$USERNAME
+
+# Setup tizen studio CLI
+RUN set -x \
+    # Generate authr certificate
+    && $TIZEN_SDK_ROOT/tools/ide/bin/tizen certificate --alias=CHIP --name=CHIP --email=chip@tizen.org --password=chiptizen \
```

Comment:
what are this username and passwords used for? Is there any concern about having these in plain text in a OSS repository?

Reply:
This generated author certificate is used to sign example application. I think that someone who will use Matter SDK to build Tizen application will not use this docker image anyway (but I might be wrong...). Alternatively, we might describe this step in the documentation (how to build Tizen example) as a prerequisite for building an app. In that case everyone will have their own author certificate.

How it works in details:

When generating certificate with Tizen Studio CLI it's required to specify password for private key. In the next line, the same password is used to "open" this certificate and add it to the so called "security profile". The security profile consists of author certificate and distributor certificate. Installed Tizen Studio uses a dummy certificate as a distributor certificate, but it's good enough to test application with Tizen image. When uploading app to the store, app is repacked with proper distributor certificate (certificate of a store) but the author certificate remains the same.